src/utils/PopoverButton.vue

```diff
@@ -136,6 +136,10 @@ export default {
             // shows an arrow at the border of the popover that points to the button location
             arrow: true,
             trigger: 'click',
+            // The default tippy maxWidth is 350px, we need to add one more pixel here in order
+            // to avoid a useless horizontal scrollbar in the drawing icon selection menu on
+            // Windows Edge Browser.
+            maxWidth: 351,
```

Comment:
in Bootstrap's variable, there's one that is called
```
$toast-max-width:                   350px !default;
```
We could maybe use it with a `calc($toast-max-width + 1);` so that we avoid having fixed value in our component's (S)CSS

Reply:
I removed and resize the icons to fix the issue, see https://github.com/geoadmin/web-mapviewer/pull/275/commits/992467c8590c4122b569241235084cf3ba505b74